### PR TITLE
MULE-14198: flow-ref is missing target attribute

### PR DESCRIPTION
**Pull request by elrodro83**

### CODE REVIEW

File: core/src/main/java/org/mule/runtime/core/internal/util/rx/Operators.java [hunk 1]
```diff
@@ -46,6 +51,19 @@ private Operators() {}
     };
   }
 
+  public static Function<CoreEvent, CoreEvent> outputToTarget(CoreEvent originalEvent, String target,
+                                                              String targetValueExpression,
+                                                              ExtendedExpressionManager expressionManager) {
+    return result -> {
+      if (target != null) {
+        TypedValue targetValue = expressionManager.evaluate(targetValueExpression, getTargetBindingContext(result.getMessage()));
+        return builder(originalEvent).addVariable(target, targetValue.getValue(), targetValue.getDataType()).build();
```
Comment (marianogonzalez) [inline core/src/main/java/org/mule/runtime/core/internal/util/rx/Operators.java]: out of this PR scope, but this api is very chota

File: core/src/main/java/org/mule/runtime/core/internal/util/rx/Operators.java [hunk 2]
```diff
@@ -46,6 +51,19 @@ private Operators() {}
     };
   }
 
+  public static Function<CoreEvent, CoreEvent> outputToTarget(CoreEvent originalEvent, String target,
+                                                              String targetValueExpression,
+                                                              ExtendedExpressionManager expressionManager) {
+    return result -> {
+      if (target != null) {
```
Comment (marianogonzalez) [inline core/src/main/java/org/mule/runtime/core/internal/util/rx/Operators.java]: @afelisatti you probably want to leverage this on your PR

File: core/src/main/java/org/mule/runtime/core/internal/util/rx/Operators.java [hunk 3]
```diff
@@ -46,6 +51,19 @@ private Operators() {}
     };
   }
 
+  public static Function<CoreEvent, CoreEvent> outputToTarget(CoreEvent originalEvent, String target,
+                                                              String targetValueExpression,
+                                                              ExtendedExpressionManager expressionManager) {
```
Comment (afelisatti) [inline core/src/main/java/org/mule/runtime/core/internal/util/rx/Operators.java]: Wouldn't `ExpressionManager` do?

Comment (elrodro83) [reply core/src/main/java/org/mule/runtime/core/internal/util/rx/Operators.java]: `ExpressionLanguage` actually

**Pull request merged**